devices/pine64-pinephone/default.nix

```diff
@@ -2,7 +2,6 @@
 
 {
   imports = [
-    ./modem.nix
```

Comment:
If we're dropping the file from imports, I assume the file should be deleted, right?

Reply:
Done.